assets/section-main-product.css

```diff
@@ -1245,6 +1245,26 @@ a.product__text {
   flex-shrink: 0;
 }
 
+/* Inventory status */
+
+.product__inventory {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.product--no-media .product__inventory {
+  justify-content: center;
+}
+
+p.visibility-hidden:empty {
+  display: block;
+}
+
+p.visibility-hidden:empty::after {
+  content: "#";
+}
```

Comment:
I think those declarations are too generic. They could be impacting other `p` elements in different part of the theme so they should be more specific to the inventory feature. Using `.product__inventory` instead of `p`.

Reply:
good call, thanks Ludo!